src/js/hooks/getResortFlagImage.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * Get the resort country flag
+ *
+ * @param {string} countryCode
+ * @return {string}
+ */
+const getResortFlagImage = (countryCode) => (countryCode ? `https://flagcdn.com/${countryCode.toLowerCase()}.svg`
```

Comment:
```suggestion
const useResortFlagImage = (countryCode) => (countryCode ? `https://flagcdn.com/${countryCode.toLowerCase()}.svg`
```

Hooks have the 'use' suffix